docs/manual/src/internals/lifting_and_lowering.md

```diff
@@ -94,3 +94,22 @@ Note that length fields in this format are serialized as *signed* integers
 despite the fact that they will always be non-negative. This is to help
 ease compatibility with JVM-based languages since the JVM uses signed 32-bit
 integers for its size fields internally.
+
+## Code Generation and the FfiConverter trait
+
+UniFFI needs to generate Rust code to lift/lower types.  To help with this, we define the `FfiConverter` trait which contains the code to lift/lower/serialize a particular type.
+
+The most straightforward approach would be to define `FfiConverter` on the type being lifted/lowered/serialized.  However, this wouldn't work for remote types defined in 3rd-part crates because of the Rust orphan rules.  For example, our crates can't implement `FfiConverter` on `serde_json::Value` because both the trait and the type are remote.
+
+To work around this we do several things:
+
+ - `FFIConverter` gets a generic type parameter.  This type is basically arbitrary and doesn't affect the lowering/lifting/serialization process.
```

Comment:
here and below, should be `FfiConverter` - we *used* to have `FFIConverter` but renamed it, right? If so, do we actually want `UniffiTag`?